src/plugins/visualizations/public/embeddable/visualize_embeddable.tsx

```diff
@@ -192,6 +192,10 @@ export class VisualizeEmbeddable
     return this.vis.description;
   }
 
+  public getVis() {
```

Comment:
Since all you need is the visualization type, maybe we could be more specific here? `getVisType` or something?

I like that better since it doesn't expose as much internal state. Let me know if you have a different opinion, though.

Reply:
Just following lens embeddable which exposes entire object, https://github.com/elastic/kibana/blob/main/x-pack/plugins/lens/public/embeddable/embeddable.tsx#L915. That way you don't have to add new methods for each piece of state needed later

Reply:
Thanks for that reference. I guess this makes sense since Typescript will catch any changes to structure of the vis object. I was looking at `getDescription` above as my guide, but that looks like a special case.